fe1-web/store/reducers/SocialReducer.ts

```diff
@@ -80,7 +124,11 @@ export const makeChirpsList = () => createSelector(
       return [];
     }
     if (chirpList.byLaoId[laoId]) {
-      return chirpList.byLaoId[laoId].allChirps;
+      const allChirps: ChirpState[] = [];
+      chirpList.byLaoId[laoId].allIdsInOrder.forEach(
+        (id) => allChirps.push(chirpList.byLaoId[laoId].byId[id]),
```

Comment:
Do you want the chirp state or the chirp itself out of the selector?

Reply:
We let the selector output chirp state, and later in SocialHome, we convert from state to the chirp object for rendering. I think we can also do the conversion inside the selector, but will it make a big difference?